### PR TITLE
fix/sidemenu-tooltips-css

### PR DESCRIPTION
**Pull request by skyrowl**
- Removed the use of Alpine on tooltips for tailwind

### CODE REVIEW

**Comment by markmead** [comment]
Haha, we did the exact same thing! #450 Let's use yours! 🥳

Can we remove the `transition opacity-0 group-hover:opacity-100` on the tooltips? It doesn't look right.

**Comment by skyrowl** [comment]
> Haha, we did the exact same thing! #450 Let's use yours! 🥳
> 
> Can we remove the `transition opacity-0 group-hover:opacity-100` on the tooltips? It doesn't look right.

Glad that we thought of the same solution !

Sure we can it was just for a smooth apparition it's not useful at all.

File: packages/admin/resources/views/livewire/sidebar.blade.php [hunk 1]
```diff
@@ -26,10 +24,7 @@ class="text-sm font-medium">
                             {{ $item->name }}
                         </span>
 
-                        <span x-cloak
-                              x-transition
-                              x-show="showTooltip"
-                              class="absolute z-10 p-2 ml-4 text-xs text-center text-white bg-gray-900 rounded dark:bg-gray-800 w-28 left-full">
+                        <span class="invisible group-hover:visible absolute z-10 p-2 ml-4 text-xs text-center text-white bg-gray-900 rounded dark:bg-gray-800 w-28 left-full">
```
Comment (markmead) [inline packages/admin/resources/views/livewire/sidebar.blade.php:27]: Can we move `group-hover:visible` to the end of the class names?

File: packages/admin/resources/views/livewire/sidebar.blade.php [hunk 2]
```diff
@@ -58,10 +51,7 @@ class="text-sm font-medium">
                     {{ __('adminhub::global.settings') }}
                 </span>
 
-                <span x-cloak
-                      x-transition
-                      x-show="showTooltip"
-                      class="absolute z-10 p-2 ml-4 text-xs text-center text-white bg-gray-900 rounded dark:bg-gray-800 w-28 left-full">
+                <span class="invisible group-hover:visible absolute z-10 p-2 ml-4 text-xs text-center text-white bg-gray-900 rounded dark:bg-gray-800 w-28 left-full">
```
Comment (markmead) [inline packages/admin/resources/views/livewire/sidebar.blade.php:54]: And the same here, please.

**Review by markmead — COMMENTED** [review]
Looks good to me! 🚀

Just two tiny changes that will save my OCD from keeping me up at night 😅

**Review by markmead — APPROVED** [review]
Happy with this! Any thoughts @alecritson?

**Comment by alecritson** [comment]
@markmead nothing from me. @glennjacobs If you're happy, seems like a small tweak.

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**